playbooks/karrot-world/setup.playbook.yml

```diff
@@ -0,0 +1,293 @@
+---
+- name: karrot.world | setup
+  hosts: all
+  become: yes
+  vars_files:
+    - secrets.vars.yml
+  vars:
+    site: karrot-world
+    server_name: karrot.world
+    redirects:
+      - alpha.foodsaving.world
+      - app.foodsaving.world
+      - www.karrot.world
+      - fstool.yunity.org
+  tasks:
+
+    - name: check ansible version
+      include_role:
+        name: check-ansible-version
+
+    - name: basic-site
+      include_role:
+        name: basic-site
+
+    - name: karrot-backend
+      include_role:
+        name: karrot-backend
+      vars:
+        site_name: karrot.world
+        static_root: /home/deploy/karrot-frontend/release
+        from_email_address: karrot.world <noreply@karrot.world>
+        redis_db: 2
+        influxdb_database: fstool
+        postgresql_database: fstool
+        postgresql_user: fstool
+        workers: 4
+
+    - name: karrot-backups
+      include_role:
+        name: karrot-backups
+      vars:
+        postgresql_database: fstool
+        gpg_id: B88A03E5
+        offsite_server: tiltec@81.30.158.213:backup/
+        gpg_secret_key: !vault |
```

Comment:
I couldn't think of a better way than to include the secret key inline (also not sure if it works).